docs/content/technical-specifications.md

```diff
@@ -17,7 +17,20 @@ The 1.12 release supports the Ingress v1beta1 API and continues to receive secur
 
 ## Supported Kubernetes Versions
 
-We explicitly test the NGINX Ingress Controller on a range of Kubernetes platforms at each release, and the [release notes](/nginx-ingress-controller/releases) list which platforms were tested. We will provide technical support for the NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider and which passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
+We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes platforms at each release, and the [release notes](/nginx-ingress-controller/releases) list which platforms were tested. We will provide technical support for the NGINX Ingress Controller (NIC) on any Kubernetes platform that is currently supported by its provider and which passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
+
+{{% table %}}
+| NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
+| --- | --- | --- | --- | --- |
+| 2.1.1 | 1.23 - 1.19 | 0.12.1 | 0.5.0 | 1.21.6 / R26 |
```

Comment:
```suggestion
| 2.1.1 | 1.23 - 1.19 | 0.12.1 | 0.5.1 | 1.21.6 / R26 |
```

Reply:
Please just maintain and fix the document.  It is far more efficient than continually returning to me when something changes since it has not been merged yet.  Or take the PR and maintain it please.